[2025-03-27T20:29:23.908+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:29:16.958415+00:00 [queued]>
[2025-03-27T20:29:23.937+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:29:16.958415+00:00 [queued]>
[2025-03-27T20:29:23.938+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-27T20:29:23.972+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transfer_data> on 2025-03-27 19:29:16.958415+00:00
[2025-03-27T20:29:23.980+0100] {standard_task_runner.py:60} INFO - Started process 24410 to run task
[2025-03-27T20:29:23.987+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag2', 'transfer_data', 'manual__2025-03-27T19:29:16.958415+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/complete.py', '--cfg-path', '/tmp/tmp6terqg6_']
[2025-03-27T20:29:23.993+0100] {standard_task_runner.py:88} INFO - Job 117: Subtask transfer_data
[2025-03-27T20:29:24.117+0100] {task_command.py:423} INFO - Running <TaskInstance: test_dag2.transfer_data manual__2025-03-27T19:29:16.958415+00:00 [running]> on host LAPTOP-A4DHQTS9.
[2025-03-27T20:29:24.335+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag2' AIRFLOW_CTX_TASK_ID='transfer_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T19:29:16.958415+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-27T19:29:16.958415+00:00'
[2025-03-27T20:29:24.375+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-27T20:29:24.400+0100] {sql.py:450} INFO - Running statement: SELECT COALESCE(MAX(lastupdated), '1900-01-01'::timestamp) FROM staging.stg_prog3;, parameters: None
[2025-03-27T20:29:24.404+0100] {sql.py:459} INFO - Rows affected: 1
[2025-03-27T20:29:24.406+0100] {logging_mixin.py:188} INFO - Latest lastupdated in staging: 1900-01-01 00:00:00
[2025-03-27T20:29:24.432+0100] {base.py:83} INFO - Using connection ID 'source_db' for task execution.
[2025-03-27T20:29:24.450+0100] {sql.py:450} INFO - Running statement: 
        SELECT * FROM prog3 
        WHERE lastupdated > '1900-01-01 00:00:00';
    , parameters: None
[2025-03-27T20:29:24.457+0100] {sql.py:459} INFO - Rows affected: 30
[2025-03-27T20:29:24.481+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-27T20:29:24.523+0100] {sql.py:547} INFO - Done loading. Loaded a total of 30 rows into staging.stg_prog3
[2025-03-27T20:29:24.523+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-03-27T20:29:24.556+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_dag2, task_id=transfer_data, execution_date=20250327T192916, start_date=20250327T192923, end_date=20250327T192924
[2025-03-27T20:29:24.602+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-27T20:29:24.658+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
